README.md

```diff
@@ -252,6 +252,32 @@ make container
 Simply build and run kube-state-metrics inside a Kubernetes pod which has a
 service account token that has read-only access to the Kubernetes cluster.
 
+### Prometheus Operator
```

Comment:
```suggestion
### For the Users of Prometheus Operator
```

Reply:
This should be maybe: 

For the users of prometheus-operator/kube-prometheus stack?

Reply:
Sounds good!

Reply:
Fixed: https://github.com/kubernetes/kube-state-metrics/pull/1412/commits/c27da11e3ba5c11a9c0e0bed0cd53608bd7cb5aa